gst/mqtt/mqttcommon.h

```diff
@@ -21,4 +21,30 @@
 #define GST_MQTT_ELEM_NAME_SINK "mqttsink"
 #define GST_MQTT_ELEM_NAME_SRC "mqttsrc"
 
+#define GST_MQTT_LEN_MSG_HDR    512
+
+/**
+ * @brief GST_BUFFER_MEM_MAX in gstreamer/gstbuffer.c is 16. To represent each
+ *        size of the memory block that the GstBuffer contains, GST_MQTT_MAX_NUM_MEMS
+ *        should be 16.
+ */
+#define GST_MQTT_MAX_NUM_MEMS   16
```

Comment:
Then, ```#define GST_MQTT_MAX_NUM_MEMS (NNS_TENSOR_SIZE_LIMIT)```

Reply:
> Then, `#define GST_MQTT_MAX_NUM_MEMS (NNS_TENSOR_SIZE_LIMIT)`

Um....libgstmqtt does not have any dependencies on NNStreamer.

Reply:
If you want to make it fixed regardless of ```GST_BUFFER_MEM_MAX``` because this is rather a network protocol, you may

Version 2021-2Q (1.0?)
: generate ```#warn``` or ```#error``` if ```GST_BUFFER_MEM_MAX > 16```

Version LATER (2.0?)
: update the protocol so that tensors[N > 16] is supported without changing the structure.

Reply:
> If you want to make it fixed regardless of `GST_BUFFER_MEM_MAX` because this is rather a network protocol, you may
> 
> Version 2021-2Q (1.0?)
> : generate `#warn` or `#error` if `GST_BUFFER_MEM_MAX > 16`
> 
> Version LATER (2.0?)
> : update the protocol so that tensors[N > 16] is supported without changing the structure.

GST_BUFFER_MEM_MAX is a static variable we cannot see. Is it possible?

Reply:
Aw... then.. yes.. let's simply leave a comment that 16 is required to be consistent with GST_BUFFER_MEM_MAX of gstbuf.c